wagtail/blocks/base.py

```diff
@@ -564,14 +564,11 @@ def render_with_errors(self, name, value, attrs=None, errors=None, renderer=None
             error = errors.as_data()[0]
             error_json = json.dumps(get_error_json_data(error))
         else:
-            error_json = "null"
+            error_json = json.dumps({})
```

Comment:
```suggestion
            error_json = "null"
```

This could change back to null or 

```suggestion
            error_json = json.dumps(None)
```